api/payara-api/src/main/java/fish/payara/security/annotations/AdfsAuthenticationDefinition.java

```diff
@@ -0,0 +1,377 @@
+/*
```

Comment:
This file is a copy-paste of another one. This needs to be inherited from a common class